README.md

```diff
@@ -3,136 +3,29 @@
-For more information, see the command help output available via either of the following:
-
-- `mbt [command] --help` 
-- `mbt [command] -h`
-
-
-### Makefile
-The `Makefile.mta` file that is generated by the `mbt init` command is the actual project "builder". It provides the verbose build manifest, which can be changed according to the project needs. It is responsible for:
-- Building each of the modules in the MTA project.
-- Invoking the MBT commands in the right order.
-- Archiving the MTA project.<br><br>
-Use the `make` command to package the MTA project with the following parameters:
-
-
-
-| Parameter        | Type | Mandatory&nbsp;/<br>Optional        | Description                 | Examples&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;    
-| -----------  | ------ | -------       |  ----------                              |  --------------------------------------
-| `p`  | string     | Mandatory     | The name of the deployment platform. <br>The supported deployment platforms are: <ul><li>cf <li>neo<li>xsa                                      |`make -f Makefile.mta p=cf`
-| `t`    | string     | Optional  | The folder for the generated MTAR file. The default value is the current folder. If this parameter is not provided, the MTAR file is saved in the `mta_archives` subfolder of the current folder. If the parameter is provided, the MTAR file is saved in the root of the folder provided by the argument.                              | `make -f Makefile.mta p=cf t=C:\temp`
-| `mtar`    | string     |   Optional  | The file name of the generated archive file. If this parameter is omitted, the file name is created according to the following naming convention: <br><br> `<mta_application_ID>_<mta_application_version>.mtar` <br><br> If the parameter is provided, but does not include an extension, the `.mtar` extension is added. | `make -f Makefile.mta p=cf mtar=myMta`<br><br> `make -f Makefile.mta p=cf mtar=myMta.mtar`
-| `strict`    | Boolean     | Optional    | The default value is `true`. If set to `true`, the duplicated fields and fields that are not defined in the `mta.yaml` schema are reported as errors. If set to `false`, they are reported as warnings. | `make -f Makefile.mta p=cf strict=false`
-
-
-#### Quick start example
-
-```go
-// Generate Makefile.mta file.
-mbt init 
+#### Multi-Target Application
 
-// Execute MTA project build.
-make -f Makefile.mta p=cf
+Before using this package, be sure you are familiar with the multi-target application concept and terminology. For background and detailed information, see the [Multi-Target Application Model](https://www.sap.com/documents/2016/06/e2f618e4-757c-0010-82c7-eda71af511fa.html) guide. 
```

Comment:
Before using this package, make sure that you are familiar...

Reply:
Updated

Reply:
Thanks